{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -13,6 +13,7 @@
 IN_DEV = ENVIRONMENT == "development"
 IN_STAGING = ENVIRONMENT == "staging"
 IN_PROD = ENVIRONMENT == "production"
+IS_REVIEW_APP = config("HEROKU_PR_NUMBER", default=0) # 0 here will result in false PR numbers start 1+
```

Comment:
You had this as a bool before. Reason for changing?

Reply:
the cast was failing !